cluster/gce/gci/configure.sh

```diff
@@ -74,6 +74,27 @@ for k,v in yaml.load(sys.stdin).iteritems():
   )
 }
 
+function download-tenantpartition-kubeconfigs {
+  local -r dest="$1"
+  local -r tp_num="$2"
+
+  echo "Downloading tenant partition kubeconfig file, if it exists"
```

Comment:
Is the kubecofig that is downloaded here different from the one generated after calling kube-up?

Reply:
they are the same. any reason this cause you think they are different ? i might be address or comment to make it clear.

Reply:
resolve as we have discussed and explained this in our meetings.